### PR TITLE
Create msft t-shirt engage form

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- [List of work items including drive-bys.]

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/engage/msvoss-win-t-shirt
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Content is roughly here: https://docs.google.com/forms/d/1JJsu__wQregWFUzJNVWOfetSi6i85nqSjAyaDukFPHo/prefill and from an email

> H1 Win an Ubuntu t-shirt at the Microsoft European Open Source Virtual Summit (can be shortened to MSVOSS)

> 
> Rest of the copy (credits to @Anastasia Valti )
> https://docs.google.com/forms/d/e/1FAIpQLSexqEVi8AG16ExVSa2K4sAZY7SzSXqQwUkkh9mcveJES1Qe8Q/viewform
> 
> With a link to our T&Cs: 
> I agree to receive information about Canonical’s products and services.
> In submitting this form, I confirm that I have read and agree to Canonical’s Privacy Notice and Privacy Policy.
> 
> Thank-you message: Thanks for participating, we will be in touch shortly
> 
> 
> I would say logo wise we can use this image:
> https://res.cloudinary.com/canonical/image/fetch/q_auto,f_auto,w_620/https://admin.insights.ubuntu.com/wp-content/uploads/517d/Ubuntu_Microsoct_Open_Source_Summit.png
> Or aven add the video from Vimeo:
> https://vimeo.com/427394909

### CODE REVIEW

File: templates/engage/msvoss-win-t-shirt.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      <img src="https://admin.insights.ubuntu.com/wp-content/uploads/517d/Ubuntu_Microsoct_Open_Source_Summit.png" width="413" height="233" alt="">
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-shallow">
+  <div class="row">
+    <div class="col-7">
+      <h4>
+        An exclusive treat for joining our booth at the Microsoft Virtual Open Source Summit: Win a first print Ubuntu 20.04 T-shirt!
```
Comment (anthonydillon) [inline templates/engage/msvoss-win-t-shirt.html:34]: I would strip the `!`

Comment (anthonydillon) [reply templates/engage/msvoss-win-t-shirt.html:34]: Please make a `<p class="p-heading--four">`

File: templates/engage/msvoss-win-t-shirt.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      </h4>
+      <p>
+        Register until 16 June 2020, 8pm (CEST) to enter the draw, and get a chance to win an Ubuntu 20.04 T-shirt before the rest of the world, in under 30 seconds.
```
Comment (anthonydillon) [inline templates/engage/msvoss-win-t-shirt.html:37]: Today?!?

Comment (pmahnke) [reply templates/engage/msvoss-win-t-shirt.html:37]: yes, emergency form

**Pull request merged**